app/models/sequencescape_submission.rb

```diff
@@ -62,7 +64,31 @@ def assets=(asset_uuids)
   # @return [Array<String>] Array of asset uuids to submit
   #
   def assets
-    @asset_groups.pluck(:assets).flatten
+    @asset_groups.pluck(:assets)
+  end
```

Comment:
Could getting rid of the `flatten` be problematic? Do you understand what it was needed for?

Reply:
Yes, I think I removed it by mistake, so rolling it back.